src/branch/option.rs

```diff
@@ -125,3 +126,12 @@ pub struct Merge {
     #[arg(long)]
     pub no_apply: bool,
 }
+
+/// Prints the current branch.
+#[derive(clap::Args, Clone, Debug)]
+pub struct Current {
+    /// Print as plain output to stdout; printing nothing instead of erroring if the current branch
+    /// can't be resolved
```

Comment:
```suggestion
    /// Print as plain text output to stdout. Prints nothing instead of erroring if the current branch
    /// can't be resolved.
```